### DISCUSSION TITLE
Can't create category

### DISCUSSION THREAD

**Discussion by chapdel** [shopperlabs/framework#101] [General] [answered]
Unable to access the creation page of a category

### Environment
php: 8.1
mysql: mysql 5.7.33
os: Windows 10

![Capture d’écran (1)](https://user-images.githubusercontent.com/20987014/161381399-b6fdd6eb-262b-4c25-941c-0ae7be6fb505.png)

**Comment by mckenziearts** [comment, marked as answer]
Your MySQL version is not compatible, please check others server requirements here https://laravelshopper.dev/requirements. For another problem please create an issue not a discussion. Thank

**Reply by chapdel** [reply]
Thank
